src/js/components/Table/Table.js

```diff
@@ -151,13 +151,15 @@ class Table extends PureComponent {
   }
 
   render() {
-    const { isStriped, className, mods, style, otherProps } = this.props
+    const { isStriped, className, mods, style, otherProps, tableBodyHeight } = this.props
 
     return (
       <Panel className={ className } mods={ mods } isStriped={ isStriped } style={ style } { ...otherProps }>
         <PanelBody>
           <PanelRow isWithCells>{ this.renderTableColumns() }</PanelRow>
-          { this.renderTableRows() }
+          <div style={{height: tableBodyHeight, overflow: 'scroll' }}>
```

Comment:
We may want to only add this wrapper and styles if we want the body to scroll. Sometimes the `overflow` property can act in unexpected ways.

Reply:
yeah, might have to add this as some kind of prop.  probably default to not setting the overflow height?  🤔